homeassistant/components/sensor/bom.py

```diff
@@ -202,13 +234,13 @@ def update(self):
 
```

Comment:
Not really part of this PR, but in the if statement above we check if the last update was within 35 minutes.

Can't we just set `MIN_TIME_BETWEEN_UPDATES` to 35 minutes? The `Throttle` should take care of it. I'm a bit confused.

Reply:
SGTM I'll raise another PR